src/hotspot/share/gc/z/z_globals.hpp

```diff
@@ -104,6 +104,13 @@
   product(bool, ZVerifyRemembered, trueInDebug, DIAGNOSTIC,                 \
           "Verify remembered sets")                                         \
                                                                             \
+  product(bool, ZUseMediumPageSizeRange, true, DIAGNOSTIC,                  \
+          "Allow multiple medium pages sizes")                              \
```

Comment:
Maybe skip the s to match the flag ?
```suggestion
          "Allow multiple medium page sizes")                              \
```

Reply:
```suggestion
          "Allow multiple medium page sizes")                               \
```